sample/tox.ini

```diff
@@ -0,0 +1,17 @@
+; tox docs: https://tox.wiki/en/latest/index.html
+
+[tox]
+envlist = py310
+skipsdist = True
+
+[testenv]
+deps = -rrequirements.txt
+
+[testenv:test]
+commands =
+    pytest -vv tests/
+
+[testenv:cowsay]
+deps = cowsay
```

Comment:
cowsay is not installed by default in most linux distributions, it is also not available by default in Windows

Reply:
Changed to 3 unit tests.